python/dicebeard/__init__.py

```diff
@@ -97,16 +102,18 @@ def __init__(self, *args, **kwargs):
             return
 
         # TODO removed image use while we fix a bug with image processing.
-        # i = no_of_dice
-        # out, total = self.my_dice.train(i)
-        # await self.sender.sendPhoto(out)
+        i = no_of_dice
+        out, total = self.my_dice.train(i)
+        await self.sender.sendPhoto(out)
         # TODO replace this dice hacking with something better like pydice
-        dice = [random.randint(1,6) for x in range(no_of_dice)]
-        total = sum(dice)
-        await self.sender.sendMessage(str(dice))
-
+        #dice = [random.randint(1,6) for x in range(no_of_dice)]
+        #total = sum(dice)
+        #await self.sender.sendMessage(str(dice))
+        
+        my_listener = self.bot.create_listener() 
+        my_listener.capture([{'from':{'id':msg['from']['id']}}])
```

Comment:
Easy enough to fix. I was running on the assumption the new listener worked the same as the one that is automatically created i.e. self.listener, which only listens to the current chat.